src/ir/context.rs

```diff
@@ -630,15 +634,29 @@ impl<'ctx> BindgenContext<'ctx> {
                 // argument names don't matter in the global context.
                 if (declaration.kind() == CXCursor_ClassTemplate ||
                     declaration.kind() ==
-                    CXCursor_ClassTemplatePartialSpecialization) &&
+                    CXCursor_ClassTemplatePartialSpecialization ||
+                    declaration.kind() == CXCursor_TypeAliasTemplateDecl) &&
                    *ty != canonical_declaration.cur_type() &&
                    location.is_some() &&
                    parent_id.is_some() {
+                    // For type aliases, there's no way to get the template
+                    // parameters as of this writing if it's a specialization,
+                    // so explicitly return `None` so we resolve the canonical
+                    // type if there is one and it's exposed.
+                    //
+                    // This is _tricky_, I know :(
+                    if declaration.kind() == CXCursor_TypeAliasTemplateDecl &&
+                       ty.canonical_type().is_valid_and_exposed() {
+                        println!("{:?}", ty.canonical_type());
```

Comment:
Use `warn!` or `debug!` instead of `println!`. Also, a diagnostic message is needed.